File: src/api/DnsWatcher.Domain/Entities/Servers/DnsServer.cs
using DnsWatcher.Domain.Common;

namespace DnsWatcher.Domain.Entities.Servers
{
	public class DnsServer : EntityBase
	{
		public string Name { get; set; }
		public string IpAddress { get; set; }
		public int Port { get; set; } = 53;
	}
}

File: src/pwa/src/app/main/models/entities/servers/dns-server.ts
export class DnsServer {
  id?: string;
  modifiedOn?: Date;

  name?: string;
  ipAddress?: string;
  port?: number;

  constructor(server: DnsServer | null = null) {
    if (server) {
      Object.assign(this, server);
    }
  }

  get pretty(): string {
    return `${this.name} (${this.ipAddress}:${this.port})`;
  }
}
